hafidh_controller:
  type: hafidh_controller/HafidhController
  arm_id: panda